config/default.yml

```diff
@@ -10,6 +10,20 @@ AllCops:
     - spec/factories/**/*.rb
     - features/support/factories/**/*.rb
 
+RSpec/AggregateExamples:
+  Description: Checks if example groups contain more than one aggregateable example.
+  Enabled: false
+  StyleGuide: http://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/AggregateExamples
+  AddAggregateFailuresMetadata: false
+  MatchersWithSideEffects:
+  - allow_value
+  - allow_values
+  - validate_presence_of
+  - validate_absence_of
+  - validate_length_of
+  - validate_inclusion_of
+  - validates_exclusion_of
```

Comment:
Agree, but honestly, I'm not sure how to make this safe and not tiresome to configure and by default at the same time.

A couple of arguments for keeping this in the default config:
 - this is a generic cop, but it also supports Rails. It could probably inject those shoulda-matchers to config, but that might not be desirable, say if side effects are eventually fixed in some version of it. Don't see a good solution to this
 - shoulda-matchers also [support non-Rails](https://github.com/thoughtbot/shoulda-matchers#non-rails-apps) that use ActiveModel or ActiveRecord
 - we already have `Rails/HttpStatus` in our default config, which is for one of the few `rspec-rails` matchers. I don't think there's a critical mass yet to extract those to a separate gem